src/TLuaInterpreter.cpp

```diff
@@ -3265,202 +3265,322 @@ int TLuaInterpreter::setBackgroundImage( lua_State *L )
 
 int TLuaInterpreter::setLabelClickCallback( lua_State *L )
 {
-    string luaSendText="";
-    if( ! lua_isstring( L, 1 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if(! pHost) {
+        lua_pushstring( L, tr( "setLabelClickCallback: NULL Host pointer - something is wrong!" )
+                        .toUtf8().constData() );
         lua_error( L );
         return 1;
     }
-    else
-    {
-        luaSendText = lua_tostring( L, 1 );
-    }
-    string luaName="";
-    if( ! lua_isstring( L, 2 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+
+    QString labelName;
+    if( ! lua_isstring( L, 1 ) ) {
+        lua_pushstring( L, tr( "setLabelClickCallback: bad argument #1 type (label name as string expected, got %1!)" )
```

Comment:
From your screenshot, the messages at the top about the lua api probably shouldn't be there, but they shouldn't be translated because they are messages for developers. If you aren't going to write code, then you don't really care if the sqlite3 module is loaded or not.

The green messages about the connection should get translated because they are in fact, user messages that the user wants to see.

Everything after that is a developer running lua code and all of those should be in English.